dist/restore/index.js

```diff
@@ -2831,18 +2836,29 @@ function run() {
             try {
                 const cacheEntry = yield cacheHttpClient.getCacheEntry(keys);
                 if (!((_a = cacheEntry) === null || _a === void 0 ? void 0 : _a.archiveLocation)) {
-                    core.info(`Cache not found for input keys: ${keys.join(", ")}.`);
+                    core.info(`Cache not found for input keys: ${keys.join(", ")}`);
```

Comment:
Note that this change was originally made in #195 but the Action wasn't recompiled.